client/src/Views/Landing/Landing.tsx

```diff
@@ -1,12 +1,21 @@
-
+import "./Landing.css";
+import logo from "../../assets/logo_bluegrey.png";
+import SingUp from "../../Components/ButtonSingUp/SingUp";
 
 const Landing = () => {
   return (
-    <div>
-      <p>landing</p>
+    <div className="landing flex justify-center text-center items-center mt-36">
+      <div className="flex flex-col items-center">
```

Comment:
In this case you have a single <div> enclosing another, it would be better to have only one div

Reply:
Solucionado